lib/index.js

```diff
@@ -63,7 +62,6 @@ export {
   // form components
   DateTimeModal,
   DateTimePreview,
-  DateTimeSelector,
```

Comment:
Hmm, do we actually want to remove this component here? Or do we want to export the styled date time selector in its place? Just trying to think through how this library and its exports might continue to be used.

Reply:
Good question. @evansiroky what is your take?

Reply:
Seems like it should be in the index file. But, our only real use of otp-react-redux is via trimet-mod-otp so I'm not sure.

Reply:
We have been deleting the refactored components from index.js. See also here: https://github.com/opentripplanner/otp-react-redux/pull/147/files